services/QuillLMS/app/workers/pre_cache_admin_dashboards_worker.rb

```diff
@@ -13,6 +13,9 @@ def perform
 
     active_admin_ids.each do |id|
       FindAdminUsersWorker.set(queue: SidekiqQueue::DEFAULT).perform_async(id)
+      FindDistrictActivityScoresWorker.set(queue: SidekiqQueue::LOW, retry: 0).perform_async(id)
```

Comment:
Nice! I always forget you can set settings inline like this. 

We could consider adding a standard `retry` limit to the job as well (maybe 0, maybe 1?), in case these queries grow and have similar issues.

Reply:
Good idea.  I'll set to 2